### PR TITLE
Create HandleLightDarkTheme.md

### PR DESCRIPTION
**Pull request by kikisaints**
Initial PR for spec that covers the light and dark theme api and windows brushes for React Native.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/2802)

### CODE REVIEW

File: vnext/proposals/HandleLightDarkTheme.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+#### Handling a theme changed event
+In this case an app author would like to do something or preform some behavior when the app's theme has changed.
+
+> Note: AppThemeState will be a subclass of **NativeEventEmitter**.
+
+```JavaScript
+componentDidMount() {
+  Platform.currentTheme.addListener('themechanged', this.onAppThemeChanged);
```
Comment (decrowle) [inline vnext/proposals/HandleLightDarkTheme.md]: Is 'themechanged' correct or should it be 'appThemeChanged' (not sure if this is a set value or just what's in the AppTheme example in RNTester) ?

File: vnext/proposals/HandleLightDarkTheme.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+If the app author wants to switch the style of their component manually based on the system's theme (Dark or Light), they can do so with CSS style conditionals.
+
+```JavaScript
+<TextInput
+    style={[styles.inputStyle, this.state.appThemeState == AppTheme.dark ? styles.darkInput : styles.lightInput]}
+</TextInput>
+```
+
+#### Handling a theme changed event
+In this case an app author would like to do something or preform some behavior when the app's theme has changed.
```
Comment (asklar) [inline vnext/proposals/HandleLightDarkTheme.md:66]: typo: perform

File: vnext/proposals/HandleLightDarkTheme.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+#### Applying a system accent color variant
+In Windows, there are algorithmically generated accent colors - dubbed Light or Dark 1, 2, and 3. This example covers what it would look like to apply that using the windowsbrush object.
+
+```css
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 19,
+    fontWeight: 'bold',
+    color: {windowsbrush: 'SystemAccentColorLight3'}
```
Comment (asklar) [inline vnext/proposals/HandleLightDarkTheme.md:106]: if we're specifying the color, wouldn't it make more sense to expose the system brushes as color value consts in some object? e.g.

`color: WindowsBrushes.SystemAccentColorLight3`

File: vnext/proposals/HandleLightDarkTheme.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+| dark | string | A string value defining that the native Windows app is in **Dark** theme. |
+| light | string | A string value defining that the native Windows app is in **Light** theme. |
+| ishighcontrast | string | A string value defining that the native Windows app is in a **High Contrast** theme (any). |
+
+### windowsbrush
+Will only allow access/lookups to theme brushes and colors that exist in generic.xaml. Custom brushes defined at the native layer are out of scope to be used/referenced by this object.
+
+| Name | Type | Key | Key Types |
+|:---|:----:|:----|:--|
+| windowsbrush | object | string | Any of the 105 theme brushes that exists in genreic.xaml (i.e. SystemControlBackgroundBaseMediumBrush), plus the 7 system accent color variants. |
```
Comment (asklar) [inline vnext/proposals/HandleLightDarkTheme.md:153]: typo: generic.xaml

**Comment by ZihanChen-MSFT** [comment]
This has been a year, is this abondoned?

**Comment by kikisaints** [comment]
> This has been a year, is this abondoned?

Yes and no - this direct doc will become obsolete in a bit when we merge some of the API's with the [PlatformColor api](https://reactnative.dev/docs/next/platformcolor) while other parts will need to be reconciled=.

File: vnext/proposals/HandleLightDarkTheme.md (hunk 3 above)
Comment (jonthysell) [reply vnext/proposals/HandleLightDarkTheme.md:106]: I like this, the consts can still be string values, but it supports intellisense better.

File: vnext/proposals/HandleLightDarkTheme.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+| currentTheme | none | string | currentTheme is a dynamic function that returns the state of the system theme that the user or native app has set. |
+
+#### App theme enum
+Platform's ```currentTheme``` returns one of the enum values defined below.
+
+| Property | Type| Description |
+|:---|:----:|:----|
+| dark | string | A string value defining that the native Windows app is in **Dark** theme. |
+| light | string | A string value defining that the native Windows app is in **Light** theme. |
+| ishighcontrast | string | A string value defining that the native Windows app is in a **High Contrast** theme (any). |
```
Comment (jonthysell) [inline vnext/proposals/HandleLightDarkTheme.md:146]: Capitalization? `isHighContrast`?

**Pull request closed without merging**